main.sh

```diff
@@ -3,22 +3,22 @@
 #Run daos tests like IOR/MDtest and self_test(cart)
 #----------------------------------------------------
 
-#SBATCH -J test_daos1           # Job name
+#SBATCH -J daos1                # Job name
 #SBATCH -o stdout.o%j           # Name of stdout output file
 #SBATCH -e stderr.e%j           # Name of stderr error file
 #SBATCH -A STAR-Intel           # Project Name
-#SBATCH -p skx-dev              # Queue (partition) name
-#SBATCH -N 4                    # Total # of nodes
-#SBATCH -n 192                  # Total # of mpi tasks (48 x  Total # of nodes)
-#SBATCH -t 1:00:00             # Run time (hh:mm:ss)
-#SBATCH --mail-user=first.last@intel.com
+#SBATCH -p development          # Queue (partition) name
+#SBATCH -N 5                    # Total # of nodes
+#SBATCH -n 240                  # Total # of mpi tasks (48 x  Total # of nodes)
```

Comment:
Minor: Total number of cores on Frontera is 56. It will be nice to update the number 48 to 56 too. Just for the knowledge sake whoever tries to access these scripts that they can actually go beyond 48.

Reply:
Yes, good idea. Will update this.